applications/nrf_desktop/doc/cpu_meas.rst

```diff
@@ -23,8 +23,8 @@ Configuration
 *************
 
 To enable this module, use the :ref:`CONFIG_DESKTOP_CPU_MEAS_ENABLE <config_desktop_app_options>` Kconfig option.
-This option selects the :kconfig:option:`CONFIG_CPU_LOAD` option.
-The :kconfig:option:`CONFIG_CPU_LOAD` option enables the :ref:`cpu_load` library that is used to perform the measurements.
+This option selects the :kconfig:option:`CONFIG_NRF_CPU_LOAD` option.
+The :kconfig:option:`CONFIG_NRF_CPU_LOAD` option enables the :ref:`cpu_load` library that is used to perform the measurements.
```

Comment:
Does this change of option need a mention in the changelog/migration guide?

Reply:
This commit will be dropped, see https://github.com/nrfconnect/sdk-nrf/pull/21098